weblate/accounts/notifications.py

```diff
@@ -507,11 +507,11 @@ def notify_monthly(self) -> None:
 @register_notification
 class RepositoryNotification(Notification):
     actions = (
-        Change.ACTION_COMMIT,
-        Change.ACTION_PUSH,
-        Change.ACTION_RESET,
-        Change.ACTION_REBASE,
-        Change.ACTION_MERGE,
+        Change.ACTIONS.ACTION_COMMIT,
```

Comment:
Maybe we can remove the `ACTION_` prefix? That would be easier to read:

```suggestion
        Change.ACTIONS.COMMIT,
```

Maybe even directly use `ActionEvents` instead:

```suggestion
        ActionEvents.COMMIT,
```

What do you think?